src/main/java/com/blackducksoftware/integration/jira/config/controller/BlackDuckConfigActions.java

```diff
@@ -23,27 +23,21 @@
  */
 package com.blackducksoftware.integration.jira.config.controller;
 
-import java.io.IOException;
 import java.net.MalformedURLException;
 import java.net.URISyntaxException;
 import java.net.URL;
 import java.util.regex.Pattern;
 import java.util.regex.PatternSyntaxException;
 
-import org.apache.commons.lang3.StringUtils;
```

Comment:
I think we want to use `org.apache.commons.lang3`

Reply:
Good catch, I accidentally changed a bunch when I started and must've missed this one.